src/python/dxpy/templating/templates/nextflow/src/nextflow.sh

```diff
@@ -60,7 +60,33 @@ on_exit() {
   # backup cache
   echo "=== Execution complete — uploading Nextflow cache metadata files"
   dx rm -r "$DX_PROJECT_CONTEXT_ID:/.nextflow/cache/$NXF_UUID/*" 2>&1 >/dev/null || true
-  dx upload ".nextflow/cache/$NXF_UUID" --path "$DX_PROJECT_CONTEXT_ID:/.nextflow/cache/$NXF_UUID" --no-progress --brief --wait -p -r || true
+
+  if [[ -d .nextflow/cache/$NXF_UUID ]]; then
+    dx upload ".nextflow/cache/$NXF_UUID" --path "$DX_PROJECT_CONTEXT_ID:/.nextflow/cache/$NXF_UUID" --no-progress --brief --wait -p -r || true
+  else
+    echo "No nextflow cache has been generated."
```

Comment:
As per our discussion earlier today and the user doc (section "Scratch Folder and Intermediate Files") we won't be uploading cache (only the output files mentioned in the output block and the outputs mentioned with the publishDir directive will be uploaded).

Reply:
Please remove this part (line 64-67)